agent/04_agent_configure.sh

```diff
@@ -272,28 +273,62 @@ spec:
 EOF
 
 if [ ! -z "${MIRROR_IMAGES}" ]; then
-# TODO - get the mirror registry info from output of 'oc adm release mirror'
 
-    cat > "${MIRROR_PATH}/registries.conf" << EOF
+   # Set up registries.conf and ca-bundle.crt for mirroring
+   if [[ -z "${OC_MIRROR}" ]] || [[ ! "${OC_MIRROR}" == true ]]; then
```

Comment:
nit: I don't think you need both of these checks, although from my (suppressed) memory of autotools there were certain flavors of UNIX in the 1980s that did :laughing:

Reply:
Fixed